packages/cli/src/commands/setup/ui/libraries/tailwindcss.js

```diff
@@ -61,6 +61,11 @@ export const handler = async ({ force, install }) => {
 
   const packages = ['postcss', 'postcss-loader', 'tailwindcss', 'autoprefixer']
 
+  const recommendedVSCodeExtensions = [
+    'csstools.postcss',
```

Comment:
@zygopleural Just making sure that you've used this extension before and it works ok? Saw that the core of it hadn't been updated in a while: https://marketplace.visualstudio.com/items?itemName=csstools.postcss. (The last commit was two weeks ago but it was only to config files)

Reply:
Also pending Danny's suggestion here https://github.com/redwoodjs/redwood/pull/5099#discussion_r847953519,  this is GTG!

Reply:
yeah been working well for me 👍🏼